gradle.properties

```
@@ -1 +1 @@
-projectVersion=3.0.0-SNAPSHOT
+projectVersion=3.1.0.M1
```

This should be SNAPSHOT. 

```suggestion
projectVersion=3.1.0.SNAPSHOT
```